designer/client/src/actions/nk/displayProcessCounts.ts

```diff
@@ -15,11 +16,34 @@ export function displayProcessCounts(processCounts: ProcessCounts): DisplayProce
     };
 }
 
+const checkActualCounts = (processCounts: ProcessCounts, processToDisplay: Process) => {
+    const processCountsName = Object.keys(processCounts).sort((a, b) => a.localeCompare(b));
+    const nodes = processToDisplay.nodes
+        .sort((a, b) => a.id.localeCompare(b.id))
+        .filter((node, index) => node.id !== processCountsName[index]);
+    const newProcessCounts = { ...processCounts };
+    if (nodes.length !== 0 && processCountsName.length !== 0) {
```

Comment:
Why do we need this check `processCountsName.length !== 0`?

Reply:
at the moment when we have nodes, but processCounts is an empty object?